apps/cli/src/commands/profile/list.profile.ts

```diff
@@ -32,19 +35,53 @@ export default class ListProfile extends BaseCommand {
     const profiles = await fetchProfileConfig()
     const defaultProfile = profiles.default
     delete profiles.default
-
-    Logger.log('Profiles:')
-    Object.keys(profiles).forEach((profile) => {
-      if (defaultProfile === profile) {
-        Logger.log(`- ${profile} (default)`)
-      } else {
-        Logger.log(`- ${profile}`)
-      }
-
-      if (verbose) {
-        Logger.log(`  - API Key: ${profiles[profile].apiKey}`)
-        Logger.log(`  - Base URL: ${profiles[profile].baseUrl}`)
+    const table = new Table({
```

Comment:
I would keep the action function responsible for just fetching the data. Then, I would delegate the data to a `print` or similar function(check the refactored code) to actually display the formatted table.

Reply:
I was thinking to get the refactor for the next PR like this is just for the things are working, then refactor into a better structure, what are your thoughts on this plan

Reply:
There are other PRs that could utilize this feature, and that's why this becomes a thing to do in the first go. Maybe we can have a `printTable` util function that will take in an object and then format it into a table accordingly.

Reply:
you can refactor it later no need to keep this pr stuck for this, have to ship other important stuff